### PR TITLE
validator: update kube dependencies to v1.23.4

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

> /kind api-change

```release-note
Kuberenetes client dependencies are updated to v1.23 with this release
```

### CODE REVIEW

**Review by bswartz — CHANGES_REQUESTED** [review]

File: go.mod [hunk 1]
```diff
@@ -11,14 +11,12 @@ require (
...
-	golang.org/x/oauth2 v0.0.0-20201208152858-08078c50e5b5 // indirect
-	google.golang.org/appengine v1.6.7 // indirect
-	k8s.io/api v0.22.0
-	k8s.io/apimachinery v0.22.0
-	k8s.io/client-go v0.22.0
+	github.com/prometheus/common v0.28.0
+	k8s.io/api v0.23.4
+	k8s.io/apimachinery v0.23.4
+	k8s.io/client-go v0.23.4
 	k8s.io/component-base v0.22.0
```
[the remark this answers is not in the document]
Comment (humblec) [reply go.mod]: Yep,its good to keep in parity, refreshed the PR @bswartz

**Review by bswartz — APPROVED** [review]
/approve
/lgtm

**Comment by humblec** [comment]
On a refresh the `lgtm` was removed, @bswartz can you please bring back the same?

**Review by bswartz — APPROVED** [review]
/lgtm

**Pull request merged**